modules/alloy/src/main/ts/ephox/alloy/behaviour/blocking/BlockingTypes.ts

```diff
@@ -11,16 +11,19 @@ export type BlockFn = (
   state: BlockingState,
   getBusySpec: (root: AlloyComponent, behaviour: Behaviour.AlloyBehaviourRecord) => AlloySpec
 ) => void;
+
 export type UnblockFn = (component: AlloyComponent, config: BlockingConfig, state: BlockingState) => void;
 
 export interface BlockingConfigSpec extends Behaviour.BehaviourConfigSpec {
   readonly getRoot?: (component: AlloyComponent) => Optional<AlloyComponent>;
+  readonly focus?: boolean;
```

Comment:
Alloy changelog entry for this please

- [ ] Add alloy changelog entry

Reply:
Thanks. Forgot about that

Reply:
Marking as resolved, but don't forget to fix the missing line Neil mentioned.